wgpu-core/src/command/transfer.rs

```diff
@@ -1036,13 +1036,14 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             .get(destination.texture)
             .map_err(|_| TransferError::InvalidTexture(source.texture))?;
 
-        // src and dst texture format must be the same.
-        let src_format = src_texture.desc.format;
-        let dst_format = dst_texture.desc.format;
-        if src_format != dst_format {
+        // src and dst texture format must be copy-compatible
+        // https://gpuweb.github.io/gpuweb/#copy-compatible
+        if src_texture.desc.format.remove_srgb_suffix()
+            != dst_texture.desc.format.remove_srgb_suffix()
+        {
             return Err(TransferError::MismatchedTextureFormats {
```

Comment:
The error message for this likely needs updating to mention that the formats can differ in srgb-ness